src/expensive_services_detail/cost_metrics_of_expensive_services.py

```diff
@@ -83,6 +83,30 @@ def get_cost_and_usage_data(client, start, end, region, account_id):
                 f"ValueError occurred: {ve}.\nPlease check the input data format."
             )
 
+region_names = {
```

Comment:
Can we try defining these parameters in SSM? instead of defining in each lambda function?

Reply:
Yes @faheem-khau9 I tried that method and also tried to get region name through lightsail but regions were limited in lightsail. So opted this method.

Reply:
@yaseenzafar123 use this please and implement ssm in this case: https://stackoverflow.com/questions/60329379/how-to-store-get-json-to-from-aws-parameter-store